tests/acceptance/setup/test.js

```diff
@@ -2,10 +2,10 @@ import {expect} from 'chai';
 import chai from 'chai';
 import Mugshot from 'mugshot';
 import chaiMugshot from 'chai-mugshot';
+import WebdriverIOAdapter from 'mugshot-webdriverio';
```

Comment:
1. We should remove `mugshot-webdriverio` from Mugshot's production deps. The goal is to give flexibility to the users in choosing the tools they want.
2. Don't implicitly depend on another's package dependencies. They are implementation details. If Mugshot loses that dep, and it will, per 1., then your package will break.